packages/devtools_app/lib/src/flutter/split.dart

```diff
@@ -66,13 +66,22 @@ class Split extends StatefulWidget {
   /// logical pixels (dp, not px).
   static const double dividerMainAxisSize = 10.0;
 
+  static Axis axisFor(BuildContext context, double horizontalAspectRatio) {
```

Comment:
you could also add a simple test for this method with the existing tests for the Split class.

Reply:
I don't know how to properly set the MediaQuery.of(context).size in the test though, do you happen to know the easy way to do that?